### PR TITLE
Licenses

### PR DESCRIPTION
**Pull request by eleanorwilliams**
License and copyright added to the study files of 

- idr0001
- idr0002
- idr0009
- idr0010
- idr0011
- idr0013
- idr0016
- idr0018
- idr0019
- idr0020
- idr0021
- idr0023
- idr0025
- idr0026
- idr0027
- idr0030
- idr0036

Copyright only added to the following studies that already had license info
- idr0003
- idr0004
- idr0005
- idr0006
- idr0007
- idr0008
- idr0028
- idr0033
- idr0034
- idr0038

Studies that will need to be updated later
idr0012 - see which license Michael Boutros decides on
idr0017 - see which license Micheal Boutros decides on
idr0015 - will update when update the whole study file with publications etc

No testing needed as such as these changes just live in the study file.  The changes are done manually through the web ui in idr-next (also done).

### CODE REVIEW

**Comment by eleanorwilliams** [comment]
@sbesson I have made the following changes

- idr0006 - corrected the copyright holders in WebUI
- idr0011 -  added Publication authors to the WebUI and added new field "Study Publication Preprint" to both study file and WebUI and added link to Biorxiv article
- idr0012 - added the CC-BY-NC-ND license and copyright holders in study file and WebUI
- idr0017 - added the CC-BY-NC-ND license and copyright holders in study file and WebUI
- idr0026 - correct the copyright holders in WebUI
- idr0038 - added study level information in WebUI for top level experiment and the 3 individual experiment

Cache cleared for all studies in idr-next so should be visible now.

File: idr0007-srikumar-sumo/idr0007-study.txt [hunk 1]
```diff
@@ -43,7 +44,7 @@ Screen Number	1
 Comment[IDR Screen Name]	idr0007-srikumar-sumo/screenA																																
 Screen Description	"In this screen two mutant yeast strains unable to synthesize SUMO chains (pro-smt3allR and smt3allR), were crossed with 384 yeast strains expressing GFP-tagged proteins (Huh et al., 2003) previously linked to the DNA damage response (Tkach et al., 2012).  144 GFP-tagged proteins displayed a change in localization and/or intensity when expressed in the smt3allR mutant grown in rich medium."																																
 Screen Size	Plates: 12	5D Images: 3456	Planes: 6912	Average Image Dimension (XYZCT): 1346 x 1003 x 1 x 2 x 1	Total Tb: 0.2																												
-Screen Example Images	http://idr-demo.openmicroscopy.org/webclient/?show=well-37013	http://idr-demo.openmicroscopy.org/webclient/img_detail/95543/	"WT plate1;G13"																														
+Screen Example Images	http://idr-demo.openmicroscopy.org/webclient/?show=well-37013	http://idr-demo.openmicroscopy.org/webclient/img_detail/95543/	WT plate1;G13
```
Comment (jburel) [inline idr0007-srikumar-sumo/idr0007-study.txt]: https

Comment (eleanorwilliams) [reply idr0007-srikumar-sumo/idr0007-study.txt]: There will be lots of cases where we need to change http://idr-demo.openmicroscopy.org to https://idr.openmicroscopy.org.  I could go through them all now, or do as a separate PR.

Comment (sbesson) [reply idr0007-srikumar-sumo/idr0007-study.txt]:
```
$ git grep idr-demo | wc
      25    1718   17936
```

I would say separate PR please else this is becoming a moving target.

**Comment by sbesson** [comment]
Went through all the changes described in https://github.com/IDR/idr-metadata/pull/271#issuecomment-347186511. Work as expected and matching the last 2 commits.

On idr0038, the study -> project, project -> study cross-references all seem valid and matching the project IDs. These links might need to be double checked as a one-off step following the deployment of IDR 0.4.4 /cc @jburel 

Happy for this to go in and have a follow-up to review all the http / idr-demo  links and unify to https / idr.

**Pull request merged**